e2e/test/scenarios/dashboard/reproductions/26230-dashboard-sticky-filter-incorrectly-positioned.cy.spec.js

```diff
@@ -16,21 +16,9 @@ describe("issue 26230", () => {
       .findByDisplayValue("dashboard with a tall card 2")
       .should("not.be.visible");
 
-    cy.findByTestId("dashboard-parameters-widget-container").should(
-      "have.css",
-      "position",
-      "fixed",
```

Comment:
@WiNloSt CSS prop is an implementation detail and I believe it should not be tested. I think it is acceptable to check CSS when it uniquely indicates one of element states but it happens very rarely. In this case it is just sticky all the time and ultimately we are interested in the behavior of seeing filters toolbar when we scroll down — this is being verified. Wdyt?

Reply:
Sounds good to me. I know this is merged, but it'd be great to make sure `master` also doesn't have this CSS check anymore, otherwise it'd be confusing.